### DISCUSSION TITLE
Disable console log "x store installed" is not working

### DISCUSSION THREAD

**Discussion by alSergey** [vuejs/pinia#2750] [Help and Questions] [answered]
### Reproduction

https://stackblitz.com/edit/pinia-store-installed-repro?file=pages%2Findex.tsx

### Steps to reproduce the bug

1. Open StackBlitz
2. Open Preview in new tab
3. Open Browser DevTools
4. In console you can see this log "🍍 "exampleStore" store installed 🆕"

### Expected behavior

There is no message in the console that the store was successfully initialized.

### Actual behavior

There is a message in the console that the store has been successfully initialized.

### Additional information

I have the "Notify about new/deleted stores" setting turned off.

<img width="1512" alt="image" src="https://github.com/user-attachments/assets/c15f2846-c024-4a0b-87c3-bb45b7ccc0d9">

When I enable the setting, my logs start to duplicate.

<img width="1512" alt="image" src="https://github.com/user-attachments/assets/98ada2de-615f-48b9-ae30-07b0d7a507ce">

**Comment by posva** [comment, marked as answer]
I tested and the log comes from the nuxt devtools. You can see it disappears if you disable devtools. It's weird because they do not appear on the page but they still seem to be active. My guess is that the nuxt devtools pinia panel has its own setting (hidden in this case) and that you have effectively two devtools running

**Comment by alSergey** [comment]
Tried disabling nuxt devtools, logs disappeared.

<img width="1512" alt="image" src="https://github.com/user-attachments/assets/068145b6-d1b5-41bf-b94a-4ca0478ee6c5">
<img width="1512" alt="image" src="https://github.com/user-attachments/assets/c8f8f390-28d1-412c-9b23-ba2a9af8b020">

I can't find any way to disable pinia logs in nuxt devtools settings. Maybe there is some way to add this setting?

<img width="1512" alt="image" src="https://github.com/user-attachments/assets/85f80ddc-2c8b-4349-9d68-c4c54029e85f">

**Comment by alSergey** [comment]
@posva hi, is it possible to add a setting to disable logs in nuxt devtools?

**Reply by posva** [reply]
I suppose it is, you should check in nuxt-devtools source code, it's not here

**Reply by alSergey** [reply]
In the end, the problem turned out to be in vue devtools, [issue](https://github.com/vuejs/devtools-next/issues/593).
